xocto/storage/files.py

```diff
@@ -0,0 +1,15 @@
+import os
+from typing import IO
+
+XLRD_FLOAT_TYPE = 2
+
+
+def size(file: IO) -> int:
```

Comment:
This isn't the entire `files.py` file. 

We should move the entire file, not just parts of it

Reply:
Also, we should move the `test_files.py` file